### PR TITLE
Release v0.0.2

### PR DESCRIPTION
**Pull request by xhlulu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.2] - 2018-11-08
+### Added
+* Author email and improve description
+* Data section of demos readme
+* Dash, html and renderer to install requirements
```
Comment (chriddyp) [inline CHANGELOG.md]: Let's be explicit here. "Added the components "dash", "dash-html-components", and "dash-renderer" as explicit package requirements."

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-cytoscape",
-  "version": "0.0.1",
```
Comment (chriddyp) [inline package.json]: You'll need to update this one too: https://github.com/plotly/dash-cytoscape/blob/master/dash_cytoscape/package.json

Or, you can just run `build:py` which will copy `package.json` into `dash_cytoscape/package.json`

Comment (xhlulu) [reply package.json]: Yes it's changed now

File: CHANGELOG.md (hunk 1 above)
Comment (xhlulu) [reply CHANGELOG.md]: Updated!

**Comment by xhlulu** [comment]
Thank you!

**Pull request merged**